src/test/e2e/Helpers/Selectors.ts

```diff
@@ -3,6 +3,8 @@ module.exports = {
   ContinueButton: '.govuk-button',
   CommonPageTitle: '.govuk-heading-l',
   CommonPageTitleM: '.govuk-heading-m',
+  SubscriptionsButton: '#subscriptionHeaderButton',
+  SignInButton: "#signInButton",
```

Comment:
Build is complaining about these quotes, should be single

Reply:
Good spot - changes made accordingly